chat-client/src/client/mynahUi.ts

```diff
@@ -209,7 +209,9 @@ export const createMynahUi = (messager: Messager, tabFactory: TabFactory): [Myna
         const tabId = getOrCreateTabId()
         if (!tabId) return
 
-        mynahUi.addToUserPrompt(tabId, params.selection)
+        const markdownSelection = ['\n```\n', params.selection, '\n```'].join('')
```

Comment:
Just curious what does this do?

Reply:
That is to render the selected code as proper markdown code block instead of showing it as plaintext